content-src/components/NewTabPage/NewTabPage.js

```diff
@@ -42,11 +43,31 @@ const NewTabPage = React.createClass({
       this.setState({renderedOnce: true});
     }
   },
+  _hasWeightedHighligts() {
+    return this.props.WeightedHighlights.weightedHighlights && this.props.WeightedHighlights.rows.length;
+  },
+  _renderGroupedSectionTitle() {
+    let title = "Recent Activity";
+
+    if (this._hasWeightedHighligts()) {
```

Comment:
if you still need to do this, return a `hasWeightedHighlights` prop from the selector